scss/_patterns_image.scss

```diff
@@ -1,6 +1,37 @@
 @import 'settings';
 
 @mixin vf-p-image {
+  %vf-image-container--common {
+    img {
+      background-size: contain;
+    }
+  }
+
+  [class^='p-image-container'] {
```

Comment:
I'd probably make this:
```
.p-image-container,
[class^='p-image-container--'] {
```
Because using the starts-with selector on the entire Block class means it would apply to any Elements also, if we ever were to add any, and that's probably not desirable.